### PR TITLE
improve CLI for graph_split.py

### PR DESCRIPTION
**Pull request by altairwei**

### CODE REVIEW

File: cwl_utils/graph_split.py [hunk 1]
```diff
@@ -177,5 +198,17 @@ def seen_import(entry: MutableMapping[str, Any]) -> bool:
     return seen_imports
 
 
+def json_dump(entry: Any, output_file: str):
```
Comment (mr-c) [inline cwl_utils/graph_split.py:201]: > error: Function is missing a return type annotation

You can run `make mypy` or `tox -e py38-mypy` to check before pushing

**Review by mr-c — CHANGES_REQUESTED** [review]
Thanks for this!

However the tests no longer pass: https://travis-ci.com/github/common-workflow-language/cwl-utils/jobs/418190995#L533

Also,c an you run `make format`? We use `black` for Python code formatting

https://travis-ci.com/github/common-workflow-language/cwl-utils/jobs/418190995#L445

File: cwl_utils/graph_split.py (hunk 1 above)
Comment (altairwei) [reply cwl_utils/graph_split.py]: I'm sorry I didn't notice these, but where can I find a contribution guide?

**Comment by mr-c** [comment]
Thank you for the reminder @altairwei , I've added a basic contributors guide at https://github.com/common-workflow-language/cwl-utils/blob/main/CONTRIBUTING.md

**Comment by altairwei** [comment]
> Thank you for the reminder @altairwei , I've added a basic contributors guide at https://github.com/common-workflow-language/cwl-utils/blob/main/CONTRIBUTING.md

How to deal with the following errors:

```
cwl_utils/graph_split.py:17: error: Skipping analyzing 'cwlformat.formatter': found module but no type hints or library stubs
cwl_utils/etools_to_clt.py:21: error: Skipping analyzing 'cwltool.errors': found module but no type hints or library stubs
cwl_utils/etools_to_clt.py:21: note: See https://mypy.readthedocs.io/en/latest/running_mypy.html#missing-imports
cwl_utils/etools_to_clt.py:22: error: Skipping analyzing 'cwltool.expression': found module but no type hints or library stubs
cwl_utils/etools_to_clt.py:23: error: Skipping analyzing 'cwltool.utils': found module but no type hints or library stubs
Found 4 errors in 2 files (checked 11 source files)
```

**Comment by mr-c** [comment]
@altairwei I'll take care of those errors

File: cwl_utils/graph_split.py [hunk 2]
```diff
@@ -47,15 +76,16 @@ def my_represent_none(
...
         if entry_id == "main":
-            entry_id = "unpacked_{}".format(os.path.basename(sourceIO.name))
-        with open(entry_id, "w", encoding="utf-8") as result_handle:
-            yaml.main.round_trip_dump(
-                entry,
-                result_handle,
-                default_flow_style=False,
-                indent=4,
-                block_seq_indent=2,
-            )
```
Comment (mr-c) [inline cwl_utils/graph_split.py:58]: This needs to show up in `yaml_dump` for those who do not want to use `cwlformat`

Comment (altairwei) [reply cwl_utils/graph_split.py:58]: OK, I'll add an option `--pretty` to use `cwlformat`

Comment (altairwei) [reply cwl_utils/graph_split.py:58]: However, if you don't use `cwlformat`, the output will just be a messy json instead of the expected yaml format.

**Comment by mr-c** [comment]
Thank you @altairwei !

**Pull request merged**